src/test/scala/com/amazon/deequ/profiles/ColumnProfilerTest.scala

```diff
@@ -96,6 +96,7 @@ class ColumnProfilerTest extends WordSpec with Matchers with SparkContextSpec
           "String" -> 0
```

Comment:
Please explain some of the changes in this file

Reply:
Since I have added kll to the NumericalProfile, I added 'None' in Numerical Profile in some of the tests to in order to let the tests pass. The other changes are not necessary and I have reverted them back.